MapboxNavigation/GenericRouteShield.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import UIKit
+
+class GenericRouteShield: StylableView {
```

Comment:
Should this class be public, so that developers can customize the colors or roundedness?